io.botfather.Botfather.appdata.xml

```diff
@@ -0,0 +1,45 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!-- Copyright 2018 The Botfather Developers -->
+<component type="desktop-application">
+	<id>io.botfather.Botfather.desktop</id>
+	<metadata_license>CC0-1.0</metadata_license>
+	<project_license>LicenseRef-proprietary=https://botfather.io/docs/legal/tos/</project_license>
+	<url type="homepage">https://botfather.io</url>
+	<name>Botfather</name>
+	<summary>Universal automation framework</summary>
+	
+	<categories>
+    	<category>Development</category>
+    	<category>Utility</category>
+	</categories>
+
+	<description>
+		<p>With botfather you can create and run automation bots with ease.
+		Automate your favorite Android, Browser and Desktop applications using bot scripts.
+		Find scripts online or create your own using Botfathers convenient APIs.
+		Botfather has image recognition, custom types and many more already built in.</p>
+	</description>
+
+	<screenshots>
+		<screenshot type="default">
+			<caption>The Botfather main window</caption>
+			<image>https://botfather.io/static/metadata/flatpak/main-window.png</image>
+		</screenshot>
+		<screenshot>
+			<caption>Android devices dialog</caption>
+			<image>https://botfather.io/static/metadata/flatpak/android-window.png</image>
+		</screenshot>
+		<screenshot>
+			<caption>Window showing the integrated browser</caption>
+			<image>https://botfather.io/static/metadata/flatpak/browser-window.png</image>
+		</screenshot>
+	</screenshots>
+
+	<content_rating type="oars-1.0">
+		<content_attribute id="money-purchasing">mild</content_attribute>
```

Comment:
So does it ask you to pay on first start or what?

Reply:
No. One can use the software without paying for it.
To unlock extra functionality one has to log in using an account created on our website, which is eventually linked to Patreon.